utmap-server/src/app/main/service/eventService.py

```diff
@@ -1,6 +1,80 @@
 from flask import jsonify
 from app.main.model.event import Event
 
+def addEvent(data):
+    event = Event()
+    events = event.connectToEvents()
+    ev = events.findEvById(data['_id'], event)
+
+    if ev:
+        responseObject = {
+            'status': 'fail',
+            'message': 'Item already exists'
+        }
+        return jsonify({'result' : responseObject, 'status' : 409})
+    else:
+        newEvent =  Event(
+            name = data['name'],
+            organizer = data['organizer'],
+            startTime = data['startTime'],
+            endTime = data['endTime'],
+            building  = data['building'],
+            room = data['room'],
+            description = data['description']
+        )
+        newEvent.assignEventId(events)
+        responseBody = newEvent.formatAsResponseBody(data['building'])
+
+        responseObject = {
+            'status': 'success',
+            'message': 'Event successfully added',
+            'body': responseBody
+        }
+        return jsonify({'result': responseObject, 'status' : 201})
+
+def updateEvent(_id, data):
+    event = Event()
+    events = event.connectToEvents()
+    ev = events.findEvById(_id, event)
+
+    if ev:
+        eventToUpdate = Event(    
+            _id = _id,        
+            name = data['name'],
+            organizer = data['organizer'],
+            startTime = data['startTime'],
+            endTime = data['endTime'],
+            building  = data['building'],
+            room = data['room'],
+            description = data['description'])
+        eventToUpdate.updateEvent(events, ev)
+        responseBody = eventToUpdate.formatAsResponseBody(data['building'])
```

Comment:
Have you modified .formatAsResponseBody() yet to take a building as an argument?  Because if not, I don't believe this method will work.  Send me a screenshot of .formatAsResponseBody() if possible.